File: java/PML/src/tags/TagHeading.java
package tags;

import java.util.ArrayList;

import parser.PMLParser;

public class TagHeading extends Tag{

	@Override
	public void output(ArrayList<Tag> tags) {
		System.out.println("<h1>");
		PMLParser.parse(tags);
		System.out.println("</h1>");
	}
}

File: php/classes/PML/Tags/TagHeading.php
<?php namespace PML\Tags;

class TagHeading extends Tag {

    /**
     * Construct the base type, Tag
     *
     * @param Array $attributes
     * @param Array $childTags
     */
    public function __construct(Array $attributes, Array $childTags)
    {
        parent::__constrct($attributes, $childTags);
    }

    /**
     * Define some output for this tag
     *
     * @param  Array  $tags
     */
    public abstract function output(Array $tags)
    {
        echo '<h1>';
        PMLParser::parse($tags);
        echo '</h1>';
    }
}
